CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [1.1.11](https://github.com/ably/ably-php/tree/1.1.11) (2025-03-05)
+
+[Full Changelog](https://github.com/ably/ably-php/compare/1.1.10...1.1.11)
+
+**Closed issues:**
+
+- PHP 8.4 Deprecations - Implicitly marking parameter $cipherParams as nullable is deprecated [\#206](https://github.com/ably/ably-php/issues/206)
+- \[Question\] Does ably-php support Queues [\#200](https://github.com/ably/ably-php/issues/200)
```

Comment:
Is this issue one that's relevant for the changelog, as its not a bug/feature request?

Reply:
I mean, we have closed that issue and suggest library for the same. 
No related changes has been introduced as per issue, but we did suggest solution there.

Reply:
If you want, we can remove this issue, it's not directly related

Reply:
Fixed!